cmd/csaf_downloader/config.go

```diff
@@ -0,0 +1,66 @@
+// This file is Free Software under the MIT License
+// without warranty, see README.md and LICENSES/MIT.txt for details.
+//
+// SPDX-License-Identifier: MIT
+//
+// SPDX-FileCopyrightText: 2022 German Federal Office for Information Security (BSI) <https://www.bsi.bund.de>
+// Software-Engineering: 2022 Intevation GmbH <https://intevation.de>
+
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/mitchellh/go-homedir"
+)
+
+const defaultWorker = 2
+
+type config struct {
+	Directory            *string  `short:"d" long:"directory" description:"DIRectory to store the downloaded files in" value-name:"DIR"`
+	Insecure             bool     `long:"insecure" description:"Do not check TLS certificates from provider"`
+	IgnoreSignatureCheck bool     `long:"ignoresigcheck" description:"Ignore signature check results, just warn on mismatch"`
+	Version              bool     `long:"version" description:"Display version of the binary"`
```

Comment:
Is this necessary for the ini-file. The output of the version number seems to be suppressed when provided in the config. Tested with
```
Insecure = true
Version = true
Verbose = true
``` 
and a call `./csaf_downloader localhost`

Reply:
Commit e9cd31aad1e61deb4e9439cd9e3373e8f54d08b9 adds a `no-ini:"true"` for the version flag, too.

Reply:
Didn't notice the flag.